src/components/views/Dashboard.js

```diff
@@ -62,17 +62,17 @@ export default props => {
         );
     }
 
-    let maybeCategory = null;
-    const sortedCategories = bumpSort(props.categories, 'All');
-    if (sortedCategories.length > 1) {
-        maybeCategory = (
-            <aside id="category">
+    let maybeRegion = null;
```

Comment:
Not a fan of the `maybeFragment` approach though it is consistent in this project anyway. Seems a bit more readable to me to see `if (regions.length)...` later in the render. Obviously a preference though since nulls are quite valid.

Reply:
Do you mean something like [this](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)? I'll definitely think about this. Maybe this approach isn't as readable as I thought.

There's also the [react-if](https://github.com/romac/react-if) project which seems nice.